utils/sccli.sh

```diff
@@ -68,7 +68,23 @@ function usage()
 
 function openshift()
 {
+    # Default OpenShift Image and Tag
+    dockerRegistry=${DOCKER_REGISTRY:-docker.io}
+    imageName=${IMAGE_NAME:-openshift/origin}
+    imageTag=${IMAGE_TAG:-v1.1.1}
+
     clean_setup
+
+    # Setup the systemd environment - desired image may have changed
+    sudo sed -i.back "/^IMAGE=*/cIMAGE=\"${imageName}\:${imageTag}\"" /etc/sysconfig/openshift_option
+
+    # ensure the docker containers are present
+    docker pull ${dockerRegistry}/${imageName}:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-haproxy-router:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-deployer:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-docker-registry:${imageTag}
+    docker pull ${dockerRegistry}/${imageName}-sti-builder:${imageTag}
+
```

Comment:
We should also put a check to see the images are already available. If so we can skip the `docker pull`